sentry-core/src/main/java/io/sentry/core/protocol/SdkVersion.java

```diff
@@ -45,4 +47,12 @@ public void addPackage(String name, String version) {
   public void acceptUnknownProperties(Map<String, Object> unknown) {
     this.unknown = unknown;
   }
+
+  public Map<String, String> getIntegrations() {
+    return integrations;
+  }
+
+  public void setIntegrations(Map<String, String> integrations) {
```

Comment:
Since the `Scope` requires thread-safe collections, I'm not sure it's a good idea exposing this setter.
Perhaps we only expose: `setIntegration(Map.Key<String, String> integration);` and add that to the list ourselves.